pkg/minishift/clusterup/clusterup.go

```diff
@@ -65,17 +66,42 @@ func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runne
 	cmdArgs := []string{"cluster", "up", "--use-existing-config"}
 
 	fmt.Println("-- Checking `oc` support for startup flags ... ")
+
+	// Deal with extra flags
+	var extraFlags []string
+	if val, ok := clusterUpParams[startFlags.ExtraClusterUpFlags.Name]; ok {
+		extraFlags = strings.Fields(val)
```

Comment:
this is why isFlag was used... but at the moment I pass it untested to the `oc cluster up` arguments